eZ/Publish/Core/IO/Tests/UrlDecorator/PrefixTest.php

```diff
@@ -46,7 +47,12 @@ public function testUndecorate($url, $prefix, $decoratedUrl)
      */
     protected function buildDecorator($prefix)
```

Comment:
```suggestion
    protected function buildDecorator(string $prefix): UrlDecorator
```
(as a bonus you could drop PhpDoc for this method now)